spec/features/user_signs_in_spec.rb

```diff
@@ -189,27 +191,6 @@
 
   end
 
-  scenario 'with an email address and password' do
```

Comment:
it was identical to the "happy path", which i've now  renamed to "authentication using email address and password"